src/nfts/saga.ts

```diff
@@ -11,6 +14,37 @@ import { call, put, select, spawn, takeLeading } from 'typed-redux-saga'
 
 const TAG = 'NftsSaga'
 
+export async function fetchNftsForSupportedNetworks(address: string): Promise<NftWithNetworkId[]> {
```

Comment:
Should this function be tested independently with unit tests or are the tests in the saga sufficent?

Reply:
this is covered in the saga test, just a convenience function. Will drop the export